setup/install-pre-req.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#This script sets up the pre-requisites for setting up app-studio in local machine.
+# mainly setting up pip3,yq, kubectl, kustomize
+# Firsly it checks which type of system is running this
+# Then it installs the dependencies one by one
+
+OSNAME=$(grep '^NAME=' /etc/os-release)
+echo $OSNAME
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+echo $ROOT
+
+case $OSNAME in
```

Comment:
I think $OSTYPE will work better here: https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script

This calculation of $OSNAME doesn't work on a Mac.

Reply:
After determining $OSNAME is looks like you still need the $OSTYPE to distinguish between different Linux distributions.